vcf-dumper/vcf-dumper-ws/src/main/java/uk/ac/ebi/eva/vcfdump/server/HtsGetResponse.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.vcfdump.server;
+
+import org.opencb.biodata.models.feature.Region;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+class HtsGetResponse {
+
+    private String format;
+
+    private List<Map<String, String>> urls;
+
+    HtsGetResponse(String format, String host, String id, String chromosome, String species,
+                   List<Region> regions) {
+        this.format = format;
+        constructUrls(host, id, chromosome, species, regions);
```

Comment:
Just an small suggestion. 
I prefer this:
`this.urls = constructUrls(host, id, chromosome, species, regions);`
to this:
`constructUrls(...);`
because it makes clear that urls is a product of the constructUrls method